src/trossen_rslidar_node.cpp

```diff
@@ -252,6 +252,8 @@ CallbackReturn PointCloudLFNode::on_deactivate(const rclcpp_lifecycle::State & /
   RCLCPP_DEBUG(get_logger(), "Deactivating...");
 
   driver_ptr_->stop();
+  free_point_cloud_queue_.clear();
+  point_cloud_queue_.clear();
```

Comment:
I'm not sure how we can reset/reinitialize these on cleanup. These are class objects from a custom SyncQueue class inside the driver.

Reply:
Do they get rebuilt in the driver setup during on_configure or on_activate?

Reply:
Queues are created in the constructor and cleared on cleanup